src/modules/timetables/timetables.controller.ts

```diff
@@ -93,4 +94,22 @@ export class TimetablesController {
     );
     return toJsonTimetable(timeTable);
   }
+
+  @Post('/:timetableId/reorder')
+  async reorderTimetable(
+    @Param('userId') userId: number,
+    @Param('timetableId') timetableId: number,
+    @Body() body: ReorderTimetableDto,
+    @GetUser() user: session_userprofile,
+  ) {
+    // TODO: use user by auth instead of userId by endpoint param
+    // userId should be removed from endpoint in the future
+    // since each user should only control their own timetable
```

Comment:
```suggestion
    /**
     * TODO: use user by auth instead of userId by endpoint param
     * userId should be removed from endpoint in the future
     * since each user should only control their own timetable
     */
    @Param('userId') userId: number,
    @Param('timetableId') timetableId: number,
    @Body() body: ReorderTimetableDto,
    @GetUser() user: session_userprofile,
  ) {
    timetable
```
(작은 제안) 코멘트가 userId에 대한 comment인 것 같은데, userId 에 commenting이 되도록 위에 둘까요??